js/server/tests/aql/aql-view-iresearch-noncluster.js

```diff
@@ -298,10 +298,15 @@ function iResearchAqlTestSuite () {
 //    },
 
     testPhraseFilter : function () {
-      var result = AQL_EXECUTE("FOR doc IN VIEW UnitTestsView FILTER PHRASE(doc.text, 'quick brown fox jumps', 'text_en') RETURN doc", null, { waitForSync: true }).json;
+      var result0 = AQL_EXECUTE("FOR doc IN VIEW UnitTestsView FILTER PHRASE(doc.text, 'quick brown fox jumps', 'text_en') RETURN doc", null, { waitForSync: true }).json;
```

Comment:
Does the `waitForSync` attribute make a difference here?
I am asking because the query is a read-only query. Does `waitForSync` influence the behavior in any way here?

Reply:
Yes, 'waitForSync' is an internal undocumented hack used in tests to force a full flush to the IResearch datastore, so that the test doesn't have to wait for the asynchronous thread, i.e. consistent-now v.s. eventually-consistent

Reply:
alright